python-main.js

```diff
@@ -1509,19 +1521,20 @@ function web_editor(config) {
     function modalMsg(title, content, links){
         var overlayContainer = "#modal-msg-overlay-container";
         $(overlayContainer).css("display","block");
+        $(overlayContainer).css("display", "block");
         $("#modal-msg-title").text(title);
         $("#modal-msg-content").html(content); 
+        var modalLinks = [];
         if (links) {
-            var modalLinks = [];
             Object.keys(links).forEach(function(key) {
                 if (links[key] === "close") {
                     modalLinks.push('<a href="#" onclick = "$(\'' + overlayContainer + '\').hide()">Close</a>');
                 } else {
                     modalLinks.push('<a href="' + links[key] + '" target="_blank">' + key + '</a>');
                 }
             });
-            $("#modal-msg-links").html((modalLinks).join(' | '));
         }
+        $("#modal-msg-links").html((modalLinks).join(' | '));
```

Comment:
Not sure why this has been changed outside the if statement? Was it not working correctly before?

Reply:
Having it outside the if statement clears the links in the modal when it's closed. When this line is inside the if statement, the links are carried across to other modals that dont have any links.

If this modal is opened first:
<img width="1440" alt="Screenshot 2019-10-01 15 57 01" src="https://user-images.githubusercontent.com/53710678/65974043-2b8c7a80-e464-11e9-8121-b89d063c726e.png">

Then the file extension modal shows the same links:
<img width="634" alt="Screenshot 2019-10-01 15 57 13" src="https://user-images.githubusercontent.com/53710678/65974088-3e9f4a80-e464-11e9-8ec8-7284c7dcbf49.png">

Reply:
You moved the line back, does that mean this issue is now resolved as well?

Reply:
Didnt mean to move it back. Moved it to take the screenshots and forgot to change it after that.